webpack.config.js

```diff
@@ -386,6 +387,10 @@ module.exports = (env, argv) => {
                 chunks: ['usercontent'],
             }),
 
+            new HtmlWebpackInjectPreload({
+                files: [{ match: /.*\.(woff2?|ttf|eot)$/ }],
```

Comment:
Hmm, perhaps you could add `Inter` to the match expression as well, and then skip the conditional in the template? Right now we seem to have half the path in one place, and half in another.

Reply:
No doubt that the logic is a bit scattered currently.
However even if I was to change the files that `HtmlWebpackInjectPreload` matches I would still have to do some filtering. 

`HtmlWebpackPlugin` is designed at first to create a bunch of tags to inject in the `<head>` of right before the `</body>`. 

However if we look at the plugin definition for `src/vector/index.html` we can read

```javascript
{
    // ...
	// we inject the links ourselves via the template, because
    // HtmlWebpackPlugin will screw up our formatting like the names
    // of the themes and which chunks we actually care about.
    inject: false
}
```

Changing how the entry point is fundamentally generated seems out of scope for this task but I agree with you, would probably need some attention in the future